cmd/thetool/common.go

```diff
@@ -18,6 +18,7 @@ type Event struct {
 	Day         int                 `json:"day" yaml:"day"`
 	Calendar    map[string][]string `json:"calendar" yaml:"calendar"`
 	Holiday     map[string][]string `json:"holiday,omitempty" yaml:",omitempty"`
+	Sources     []string            `json:"sources" yaml:"sources"`
```

Comment:
please add omitempty to json tag that makes json file much lighter.

Reply:
The plan is to make it required. Also since its an array, the `omitempty` is almost useless.

Reply:
done